tests/unit/tasks/api/all.ts

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import nock from 'nock';
+import { createClient } from '../../../../src';
+import { TASKS_BASE } from '../../../../src/constants';
+import {
+  UsersResponse,
+  UserResponse,
+  randomUser,
+  User,
+  userData,
+} from '../../../__helpers__/api';
+import { randomHexString } from '../../../__helpers__/utils';
+
+describe('Tasks - Functions - API', () => {
+  const host = 'https://api.xxx.extrahorizon.com';
+  const name = 'test';
```

Comment:
(minor) Would `functionName` make sense here? Found it a (very) little bit confusing reading the test cases.